crates/grpc/src/uopool.rs

```diff
@@ -332,17 +353,45 @@ pub async fn uopool_service_run(
                 reputation.add_whitelist(addr);
             }
 
+            let entry_point = EntryPoint::<Provider<Http>>::new(eth_client.clone(), ep);
+
+            let mut validator =
+                StandardUserOperationValidator::new(eth_client.clone(), entry_point.clone(), chain)
+                    .with_sanity_check(SenderOrInitCode {})
+                    .with_sanity_check(VerificationGas {
+                        max_verification_gas,
+                    })
+                    .with_sanity_check(Paymaster {})
+                    .with_sanity_check(CallGas {})
```

Comment:
A nitpick. With the definition `pub struct CallGas;`, here you could just use `CallGas` instead of `CallGas {}`.